modules/mysql/charts.go

```diff
@@ -462,6 +462,34 @@ var charts = Charts{
 			{ID: "open_tables", Name: "tables"},
 		},
 	},
+	{
+		ID:    "process_list_fetch_duration",
+		Title: "Process List Fetch Duration",
+		Units: "milliseconds",
+		Fam:   "process list",
+		Ctx:   "mysql.process_fetch_query_duration",
+		Dims:  Dims{{ID: "process_fetch_query_duration", Name: "Query Duration"}},
+	},
+	{
+		ID:    "process_list_query_count",
+		Title: "Queries Count",
+		Units: "queries",
+		Fam:   "process list",
+		Ctx:   "mysql.process_list_query_count",
+		Type:  module.Stacked,
+		Dims: Dims{
+			{ID: "process_list_query_count_system", Name: "System"},
+			{ID: "process_list_query_user", Name: "Users"},
+		},
+	},
+	{
+		ID:    "process_list_longest_query_duration",
+		Title: "Longest Query Duration",
+		Units: "seconds",
+		Fam:   "process list",
+		Ctx:   "mysql.process_list_longest_query_duration",
+		Dims:  Dims{{ID: "process_list_longest_query_duration", Name: "Longest Query Duration"}},
```

Comment:
```suggestion
		ID:    "process_list_longest_query_duration",
		Title: "Longest Query Duration",
		Units: "milliseconds",
		Fam:   "process list",
		Ctx:   "mysql.process_list_longest_query_duration",
		Dims:  Dims{{ID: "process_list_longest_query_duration", Name: "duration"}},
```

⚠️ don't submit the changes via GitHub

- let's have it in "milliseconds. Netdata automatically converts seconds to "DD:HH:MM:SS" (we don't want to lose milliseconds)
- lowercase the dimension name

Reply:
The underlaying query will still report seconds but I'll convert them to milliseconds. In mariaDB we get the milliseconds `TIME_MS` but mysql only reports seconds. If I differentiate the queries then we will probably cause confusion. I suggest that we stick with the seconds `TIME` column in the query

Reply:
> The underlaying query will still report seconds

Ok, I should have paid more attention. `TIME` is "_The time in seconds that the thread has been in its current state._", let's keep it in seconds then.